client/src/api/v2/sensors/data.ts

```diff
@@ -0,0 +1,101 @@
+import { useState, useCallback } from 'react';
+import { useChannel } from '../socket';
+
+/* eslint-disable camelcase */
+export interface SensorData {
+  /** GPS sensor on */
+  gps?: number;
```

Comment:
Any reason this is a number? Probably a noob question but I would have thought it would be a boolean.

Reply:
> Any reason this is a number? Probably a noob question but I would have thought it would be a boolean.

This is remnants from the DAS Teensy Arduino C code where they don't have true or false. So `gps` was set to `0` or `1` for `false` and `true` respectively.

Reply:
ooooh makes sense